### PR TITLE
[quant][graphmodel] linear_relu

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#40021 [quant][graphmodel] linear_relu**

This replaces #36889 due to significant merge conflicts

Differential Revision: [D22087061](https://our.internmc.facebook.com/intern/diff/D22087061)

### CODE REVIEW

File: test/quantization/test_quantize_script.py [hunk 1]
```diff
@@ -1233,6 +1233,98 @@ def _test_op_impl(self, module, data, quantized_op, tracing=False, debug=False,
 
         return models[False]
 
+    @skipIfNoFBGEMM
+    def test_quantized_linear(self):
```
Comment (jerryzh168) [inline test/quantization/test_quantize_script.py:1237]: could you merge all the tests in one function? and add tests for tracing as well?

Comment (z-a-f) [reply test/quantization/test_quantize_script.py]: Is there an example of a tracing tests that I could base on?

File: test/quantization/test_quantize_script.py [hunk 2]
```diff
@@ -1233,6 +1233,60 @@ def _test_op_impl(self, module, data, quantized_op, tracing=False, debug=False,
...
+                super(ModuleLinear, self).__init__()
+                self.linear = torch.nn.Linear(30, 4).float()
+                if has_relu:
+                    if f_relu:
+                        self.relu = F.relu
+                    else:
+                        self.relu = torch.nn.ReLU()
+                else:
+                    self.relu = torch.nn.Identity()
+            def forward(self, x):
```
Comment (jerryzh168) [inline test/quantization/test_quantize_script.py:1249]: lint

File: test/quantization/test_quantize_script.py (hunk 1 above)
Comment (jerryzh168) [reply test/quantization/test_quantize_script.py]: for example: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_script.py#L1237

**Comment by jerryzh168** [comment]
CI is broken, please take a look

**Review by jerryzh168 — APPROVED** [review]
code looks good, please fix lint and CI before landing.

**Comment by z-a-f** [comment]
> CI is broken, please take a look

I am not sure why the CI is breaking -- it looks like for some reason unrelated tests get triggered

**Pull request closed without merging**